entity-framework/core/what-is-new/ef-core-10.0/whatsnew.md

```diff
@@ -74,6 +74,16 @@ var query = context.Students
 
 See [#12793](https://github.com/dotnet/efcore/issues/12793) and [#35367](https://github.com/dotnet/efcore/issues/35367) for more details.
 
+<a name="support-string-functions-with-char-args"></a>
+
+### Support for some string functions taking `char` as arguments
```

Comment:
This feels like a pretty minor improvement to me, more like a single-bullet within "query improvements" (like COALESCE->ISNULL above), rather than a full section.

Reply:
Yeah, I'd say so...